
# HG changeset patch
# User Thomas Waldmann <tw AT waldmann-edv DOT de>
# Date 1354999660 -3600
# Node ID d0567fba754edf749a62f3a31f7be5a70456b0b2
# Parent  d3090fb6624f600c64076e161a47207496ad356b
escape user- or admin-defined css url

diff -r d3090fb6624f -r d0567fba754e MoinMoin/theme/__init__.py
--- a/MoinMoin/theme/__init__.py	Sat Dec 08 21:20:24 2012 +0100
+++ b/MoinMoin/theme/__init__.py	Sat Dec 08 21:47:40 2012 +0100
@@ -687,7 +687,7 @@
         if theme:
             href = '%s/%s/css/%s.css' % (self.cfg.url_prefix_static, self.name, href)
         attrs = 'type="text/css" charset="%s" media="%s" href="%s"' % (
-                self.stylesheetsCharset, media, href, )
+                self.stylesheetsCharset, media, wikiutil.escape(href, True), )
         if title:
             return '<link rel="alternate stylesheet" %s title="%s">' % (attrs, title)
         else:

